.github/workflows/stale.yaml

```
@@ -0,0 +1,20 @@
+name: 'Label stale PR'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          stale-pr-message: 'This PR is stale because it has been open 7 days with no activity.'
+          days-before-pr-stale: 7
+          days-before-pr-close: -1
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity.'
+          days-before-issue-stale: 30
```

Since we don't use GH issues for tracking things we could stale/close issues immediately. We use JIRA for this repo. 
```suggestion
          days-before-issue-stale: 1
```

---

I was thinking about this as well. All issues except 2 had last updated more than 30 days ago, so even setting this to 30 will label all existing issues as stale :-D 

Ideally, we should try to close all of them and disable issues on this repo.